### ISSUE TITLE
[BUG]: 'numpy.ndarray' object has no attribute 'get_color' while plotting naive forecast

### ISSUE THREAD

**Issue by iamMaverick** [unit8co/darts#1051]
**Describe the bug**

While plotting a naive forecast an attributeError occured: 'numpy.ndarray object has no attribute get_color'

**To Reproduce**
Steps to reproduce the behavior, preferably code snippet.
I have a subset of len 6188 'train' on which I trained the darts.NaiveSessional model.

naive_model = models.NaiveSeasonal(K=1)
naive_model.fit(train)
naive_forecast = naive_model.predict(1)
naive_forecast.plot(label='Forecast')

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
c:\Users\Itcomplex\Documents\scripts\Predictive Analytics\Work\crypto_value_prediction_hourly.ipynb Cell 25' in <cell line: 4>()
      [2](vscode-notebook-cell:/c%3A/Users/Itcomplex/Documents/scripts/Predictive%20Analytics/Work/crypto_value_prediction_hourly.ipynb#ch0000026?line=1) naive_model.fit(train)
      [3](vscode-notebook-cell:/c%3A/Users/Itcomplex/Documents/scripts/Predictive%20Analytics/Work/crypto_value_prediction_hourly.ipynb#ch0000026?line=2) naive_forecast = naive_model.predict(1)
----> [4](vscode-notebook-cell:/c%3A/Users/Itcomplex/Documents/scripts/Predictive%20Analytics/Work/crypto_value_prediction_hourly.ipynb#ch0000026?line=3) naive_forecast.plot(label='Forecast')

File c:\Users\Itcomplex\anaconda3\envs\env\lib\site-packages\darts\timeseries.py:2450, in TimeSeries.plot(self, new_plot, central_quantile, low_quantile, high_quantile, *args, **kwargs)
   2447 kwargs["label"] = label_to_use
   2449 p = central_series.plot(*args, **kwargs)
-> 2450 color_used = p[0].get_color()
   2451 kwargs["alpha"] = alpha if alpha is not None else alpha_confidence_intvls
   2453 # Optionally show confidence intervals

AttributeError: 'numpy.ndarray' object has no attribute 'get_color'

**Expected behavior**

Should have plotted the predicted timeseries.

**System (please complete the following information):**
 - Python version: [e.g. 3.9.2]
 - darts version [e.g. 0.14.0]

**Additional context**

**Comment by hrzn** [comment]
Thanks for flagging. Indeed, there's a bug when plotting a series of length 1.

**Comment by iamMaverick** [comment]
> Thanks for flagging. Indeed, there's a bug when plotting a series of length 1.

Thank you Sir!

**Comment by martinb-bb** [comment]
Hi there, is there any resolution to this bug? Still seeing it.

![image](https://user-images.githubusercontent.com/105685594/184382253-1d80532f-e31f-4d63-834e-2a62f7119c69.png)

**Comment by hrzn** [comment]
It's not fixed yet - for now I would recommend not plotting series of length 1 (that's kind of awkward anyway, no? we might raise an exception instead of trying to plot).

**Comment by martinb-bb** [comment]
Oh I see, so the problem is plotting the prediction. I will have to come up with an elegant way avoid that issue. Thanks!

**Comment by hrzn** [comment]
If you use `historical_forecasts()` with `last_points_only=True` you should be fine.

**Comment by hrzn** [comment]
Closing as duplicate of https://github.com/unit8co/darts/issues/650

**Comment by hrzn** [comment]
Hi @martinb-bb, our current intent was not to fix it (although we could definitely improve the error message), although given how recurring of an issue this is, I'm having second thoughts. We can maybe plot it as a simple marker 'o' or something (let me know if you have thoughts on this).

**Comment by martinb-bb** [comment]
Hi @hrzn ! Yes, normally I wouldn't push on a specific bug but I have been getting many requests on providing a 1 day forecast.

A simple marker would completely suffice. ☺️ After all, it is simply only one point. 

I am also open to other options!

**Comment by hrzn** [comment]
I'm proposing something like this: https://github.com/unit8co/darts/pull/1209

**Comment by martinb-bb** [comment]
Interesting. So it will plot a single point with arrows to showcase the range? What is the series is not probablistic?

That is quite cool. 

Question: Historical forecast will remain the same correct? It works perfect with single day forecast.

**Comment by hrzn** [comment]
> Interesting. So it will plot a single point with arrows to showcase the range? What is the series is not probabilistic?

Yes the bars are meant to show the low/high quantiles, similar to series of length > 1. If the series is not probabilistic it only shows the point, without the range.



> Question: Historical forecast will remain the same correct? It works perfect with single day forecast.

Yes this PR only affects plotting, nothing else.

**Comment by martinb-bb** [comment]
This is great! Amazing PR. 

Thank you @hrzn ☺️
